pkgs/servers/misc/irrd/default.nix

```diff
@@ -134,10 +139,16 @@ py.pkgs.buildPythonPackage rec {
       sleep 1
     done
 
+    export SMTPD_HOST=127.0.0.1
     export IRRD_DATABASE_URL="postgres:///$PGDATABASE"
     export IRRD_REDIS_URL="redis://localhost/1"
   '';
 
+  postgresqlTestSetupPost = ''
+    echo "track_commit_timestamp=on" >> $PGDATA/postgresql.conf
```

Comment:
Why do we need this for our tests? Documentation only states to use this for query-only replicas? Similar to `hot_standby_feedback`?

Reply:
Log when building without this option: https://termbin.com/hosb

Reply:
Oh, I missed the comment above, makes sense.